platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollector.java

```diff
@@ -77,13 +85,20 @@
 
   private static final String TRUSTSTORE_PASSWORD_SYSTEM_PROPERTY =
       "javax.net.ssl.trustStorePassword";
+  private static final String FILE_FOUND_MSG = "A users.properties file was found!";
```

Comment:
These messages seem like they belong in a validator rather than the insecure defaults collector. Because we are creating a separate alert from the rest of the insecure defaults for this user properties check and it is also going to be deleting a file, I feel that the new user properties code should live in its own new collector class.

Reply:
Created a new UsersPropertiesCollector 👍